CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - n/a
 
+## [1.2.0] - 2019-07-21
```

Comment:
This isn't the way the changelog updates work, the idea is that you put your changes initially under "Unreleased".  Then, whoever is doing the releasing will move them under a new section with the right version number and date, at the time they do the release.

Reply:
got it. 
I thought I would push the patch with a new tag so it would be released.

Reply:
Could you please not?  Designated code owners for this repo are myself and Rajul, I think part of the responsibilities should be acting as the gatekeeper for releases.  Especially for a library which is trying to follow semver, I think at least one of the owners should understand what's being released.

Reply:
Okay.